spec/models/conversation_spec.rb

```diff
@@ -193,7 +193,7 @@
     it 'creates conversation activities' do
       conversation.update(
         status: :resolved,
-        contact_last_seen_at: Time.now,
+        contact_last_seen_at: Time.zone.now,
```

Comment:
Do we need to use `travel_to` to avoid flaky tests.

Reply:
this test is not dependant on the time value we provide here